pydough/conversion/relational_converter.py

```diff
@@ -78,7 +78,17 @@ class TranslationOutput:
     """
 
     relation: Relational
```

Comment:
As a follow-up can we rename the class `Relational` to `RelationalNode` so that there is a clarity that this is a node?

Reply:
Moreover, in the same lines, we can rename the field from `relation` to `relational_node`.